apps/docs/publisher_agent.md

```diff
@@ -29,10 +29,29 @@ You can also generate your own keys and build your own `secrets.json` file - it
 }
 ```
 
-You can run the docker container like this:
+The secrets.json file can be subsituted or combined with environment variables if preferred. If the same property has a value set in both the secrets.json and environment variable, the environment variable take priority.
+
+The available environment variables are:
+| Key | Details |
+|---------|---------|
+| STORK_EVM_PRIVATE_KEY | [EVM Private Key](../lib/signer/model.go#L7) |
```

Comment:
same comment - maybe this can just be a list of keys without the "details" column